### PR TITLE
Merge Request feature/milestone-2-components feature branch to develop branch

### PR DESCRIPTION
**Pull request by temesghentekeste**
Hi,
I started milestone 2 of the project by creating a feature branch named 'feature/milestone-2-components' where I wrote all the lines of code related to that milestone. The codebase is clean and passed all linting requirements.

As per the requirements: 
   - [ ] Components directory is created and the App, Display, ButtonPanel, and Button component are placed there
   - [ ] Each of the above component is implemented as per its need
   - [ ] Passing prop and prop validation are used whenever necessary
   
## As the latest version of React is used, some of the import statements might differ from old React versions.

Kindly request a pull request before I merge my feature branch to the develop branch.

Regards,
Temesghen T.

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
+  return (
+    <div>
+      <h1>{result}</h1>
+    </div>
+  );
+}
```
Comment (Torres-ssf) [inline src/components/Display.js:10]: - [ ] As it said in the requirements, we need to set a default value for the result props. This way, passing the props it's not going to be required. The default value should be a string.
```JS
Display.defaultProps = {
  result: '0',
};

Display.propTypes = {
  result: PropTypes.string,
};
```
Screenshot from 2021-04-08 09-01-10
Screenshot from 2021-04-08 09-01-22

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+import '../App.css';
+
+const App = () => (
+  <>
+    <Display result="0" />
```
Comment (Torres-ssf) [inline src/components/App.js]: - [ ] We do not need to pass the result props when it's 0 after setting it a default value for it.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hi @temesghentekeste :hand: 
Awesome job on this milestone so far :+1: :clap: 
But we need to make some changes before moving on.

- [ ] We can safely delete the `App.test.js` file since it's just a dummy test

Happy coding :computer: 
**Sérgio Torres**

**Comment by temesghentekeste** [comment]
Thanks, @Torres-ssf. I have addressed the issues you raised.

Kindly request another code review.

Regards,
Temesghen T

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @temesghentekeste :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**